tensorboard/plugins/hparams/backend_context.py

```diff
@@ -53,11 +53,11 @@ def __init__(self, tb_context, max_domain_discrete_len=10):
             Typically, only tests should specify a value for this parameter.
         """
         self._tb_context = tb_context
-        self._experiment_from_tag = None
+        self._experiment_from_tag = {}  # experiment ID to Experiment proto
```

Comment:
This name doesn't really make sense any more now that it's a dict, IMO.  Something like `self._cached_experiments` would be clearer.

For that matter though, this is now an ever-growing cache and I don't think it's actually needed (or at least not worth the risk of unbounded memory, even if in practice it probably wouldn't be a very fast leak).  Could we just remove it?

Reply:
True about the name, and fair enough that it can probably just be
removed. Done.